### PR TITLE
Changed Gradle Version in RosAndroid.groovy

### PR DESCRIPTION
**Pull request by rohbotics**
Changed Gradle Version in RosAndroid.groovy to allow for building in Android Studio 0.6.1

### CODE REVIEW

File: gradle_plugins/src/main/groovy/org/ros/gradle_plugins/RosAndroid.groovy [hunk 1]
```diff
@@ -26,7 +26,7 @@ class RosAndroidPlugin implements Plugin<Project> {
                 mavenCentral()
             }
             dependencies {
-                classpath 'com.android.tools.build:gradle:0.11.+'
+                classpath 'com.android.tools.build:gradle:1.12.+'
```
Comment (stonier) [inline gradle_plugins/src/main/groovy/org/ros/gradle_plugins/RosAndroid.groovy:29]: Wondering if you meant 0.12+ here...

**Comment by rohbotics** [comment]
I mean 1.12.+
That is the latest gradle version and works well with android studio.

**Comment by rohbotics** [comment]
I was mistaken between the gradle plugin version and the gradle version.
The current code works well.

**Comment by stonier** [comment]
:+1:

I was about to mention that.

**Pull request closed without merging**